pkg/accounting/accounting.go

```diff
@@ -83,6 +90,8 @@ var (
 	ErrPeerNoBalance = errors.New("no balance for peer")
 	// ErrOverflow denotes an arithmetic operation overflowed.
 	ErrOverflow = errors.New("overflow error")
+	// ErrInValue denotes an invalid value read from store
+	ErrInValue = errors.New("invalid value")
```

Comment:
consider renaming to `ErrInvalidValue` since `InValue` might be a bit misleading